lte/gateway/c/session_manager/GrpcMagmaUtils.cpp

```diff
@@ -79,3 +79,9 @@ std::string indentText(std::string basicString, int indent) {
   }
   return result;
 }
+
```

Comment:
Let's place these functions in `Utilities.cpp/h`

Reply:
Fixed as suggested